### PR TITLE
Longhorn: Update dev-v2.6 for v1.1.2 release

### PR DESCRIPTION
**Pull request by shuo-wu**
longhorn/longhorn#2724

### CODE REVIEW

File: packages/longhorn/package.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
-commit: 93f250e4a175044d34c6982ea6168b213caeee9f
+commit: ec1d661de72966b298833c9533089824bb911696
 version: 100.0.0
```
Comment (shuo-wu) [inline packages/longhorn/package.yaml:4]: Not sure if I need to bump the version here.

**Comment by jiaqiluo** [comment]
- [ ] need to update the version in https://github.com/rancher/rancher/issues/32294 once this PR is merged

File: packages/longhorn/package.yaml (hunk 1 above)
Comment (jiaqiluo) [reply packages/longhorn/package.yaml:4]: no, you do not.  100.0.0 is the version will be released

Comment (shuo-wu) [reply packages/longhorn/package.yaml:4]: Thanks for your clarification.

**Comment by jiaqiluo** [comment]
@shuo-wu Can you do the following in your brach:
- remove the 100.0.0+up1.1.1 version of the chart from `charts/` `assets/` and `index.yaml` ? 
- After removing the old chart, commit the changes
- then run `PACKAGE=longhorn make charts`? it should update the index.yaml to include the new version.

**Comment by shuo-wu** [comment]
Updated. Please take a look.

**Review by nickgerace — DISMISSED** [review]

**Review by cbron — DISMISSED** [review]
Needs conflicts rebased and @jiaqiluo review. Looks good, but we may update versioning.

**Comment by shuo-wu** [comment]
Conflict resolved.

**Review by jiaqiluo — DISMISSED** [review]

**Comment by cbron** [comment]
This has conflicts now due to the whole release-2.6 validate scenario. After that is fixed we can merge.

**Comment by jiaqiluo** [comment]
@shuo-wu  What you need to do is 
```
- undo the last commit for running make validate
- rebase your PR against the latest dev-v2.6
- fix conflicts if any
```
Then the CI should pass.

**Review by jiaqiluo — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**